_drafts/release-workflows.md

```diff
@@ -0,0 +1,309 @@
+| **merge** any branch to `beta`                      | Get the last release for commits on branch `beta`; if the last release is also present on a lower branch (it would be the first release on `beta`), increment the version based on new commits and make a prelease on `beta` channel; if the last release is a pre-release increment the prerelease version (`4.0.0-beta.1`, then `4.0.0-beta.2`) and release on `beta` channel       |
+| **push** to `alpha`                                 | Get the last release for commits on branch `alpha`; if the last release is also present on a lower branch (it would be the first release on `alpha`), increment the version based on new commits and make a prelease on `alpha` channel; if the last release is a pre-release increment the prerelease version (`5.0.0-alpha.1`, then `5.0.0-alpha.2`) and release on `alpha` channel |
+| **merge** any branch to `alpha`                     | Get the last release for commits on branch `alpha`; if the last release is also present on a lower branch (it would be the first release on `alpha`), increment the version based on new commits and make a prelease on `alpha` channel; if the last release is a pre-release increment the prerelease version (`5.0.0-alpha.1`, then `5.0.0-alpha.2`) and release on `alpha` channel |
+
+##### Pre-releases configuration
+
+```json
+{
+  "release": {
+    "branches": [
+      "master",
+      {"branch": "beta", "prerelease": true},
+      {"branch": "alpha", "prerelease": true},
+    ]
+  }
+}
+```
+
+#### Configuration
+
+##### Format
+
+Each branch is either a `String` or an `Object` with the following properties:
+
+| Option       | Description                                                                                                                                                             |
+|--------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `branch`     | **Require** Git branch name                                                                                                                                             |
+| `range`      | Accepted version range for to release from this branch. If defined the branch will be considered a LTS branch.                                                          |
+| `channel`    | Channel on which to release. Ignored for release targets that don't support channels.                                                                                   |
+| `prerelease` | Version of the pre-release done from this branch. Must be formatted `<version>-<tag>`. The version is required, the `tag` is optional. Examples: `4.0.0-beta`, `4.0.0`. |
```

Comment:
Is this option used now? I see a `preid` documented below, and the `version` is determined by the commits in the branch.